bitbots_vision/src/bitbots_vision/vision_modules/color.py

```diff
@@ -367,12 +364,12 @@ def __init__(self, config, package_path):
         super(DynamicPixelListColorDetector, self).__init__(config, package_path)
 
         # Annotate global variable. The global is needed due to threading issues
-        global dyn_color_space
-        dyn_color_space = np.copy(self.color_space)
+        global _dyn_color_space
```

Comment:
Should we mark a `global` defined variable as private? I get the idea behind it, but it still seems very confusing.

Reply:
I looked it up in the google python code guidelines and understood this...

Reply:
Okay I looked it up too and it looks okay.